### DISCUSSION TITLE
The PrestaShop 8.15 installation assistant failed to install the theme after entering MySQL information.

### DISCUSSION THREAD

**Discussion by likeMJhappy** [PrestaShop/PrestaShop#35755] [Q&A] [answered]
The PrestaShop 8.15 installation assistant failed to install the theme after entering MySQL information.PrestaShop 8.15 version installation assistant, after entering the MySQL information, can not install the theme, retry many times can not, MySQL version is 5.7.44, looking forward to your reply, attached figure above.
![A4B4819E-E425-4C54-B050-4D871C7F8A0C](https://github.com/PrestaShop/PrestaShop/assets/141382486/b380a910-c44e-42d0-9b4a-976268e3f1cf)

**Comment by MatShir** [comment, marked as answer]
Hi @likeMJhappy, 
could create an issue https://github.com/PrestaShop/PrestaShop/issues/new/choose for a bug report. The template from the bug report would help us to reproduce the issue on your side.
Thanks in advance
